docs/make.jl

```diff
@@ -1,7 +1,11 @@
 using Documenter, StatsBase, Statistics, Random
 
+# Workaround for JuliaLang/julia/pull/28625
+if Base.HOME_PROJECT[] !== nothing
+    Base.HOME_PROJECT[] = abspath(Base.HOME_PROJECT[])
+end
+
 makedocs(
-    format = :html,
```

Comment:
I think this should now be `format = Documenter.HTML()`

Reply:
That's the default now: https://juliadocs.github.io/Documenter.jl/stable/lib/public/#Documenter.makedocs